inspirehep/dojson/institutions/fields/bd1xx.py

```diff
@@ -85,42 +85,48 @@ def set_value(key, val):
 @utils.filter_values
 def address(self, key, value):
     """Address info."""
-    return {
-        'address': utils.force_list(value.get('a')),
-        'city': value.get('b'),
-        'state_province': value.get('c'),
-        'country': value.get('d'),
-        'postal_code': utils.force_list(value.get('e')),
-        'country_code': value.get('g'),
-    }
+    country_code = value.get('g')
+    if isinstance(country_code, (list, tuple)):
+        country_code = list(set(country_code))
```

Comment:
This will remove duplicates, but shuffle the list. The `list(set(...))` idiom reads nicely, but is annoying to test.